### ISSUE TITLE
Analyze pre-check-in landing page drop off rate

### ISSUE THREAD

**Issue by benbrasso-agile6** [department-of-veterans-affairs/va.gov-team#91433]
## Description 

Completion rate for pre-check-in was first noticed in the daily slack reminders. 

Then, noticed the biggest abandonment was happening on the landing page of pre-check-in. 

## Tasks

- [ ] Review GA4 to compare patterns between pre-check-in and check-in
- [ ] Review GA4 to compare patterns between pre-check-in before/after new landing page release 
- [ ] Chart out numbers in table
- [ ] Identify link clicks available on landing page and chart out, if possible. Before/after new landing page release would be beneficial. 
- [ ] Figure out how to include all of the other link clicks on our GA4 dashboards

**Comment by loripusey** [comment]
Since we are keeping an eye on Pre-Check-in, here are completions for the last 3 months:
- Completions for month of June = 104,938
- Completions for month of July = 128,125 (went up quite a bit in July)
- Completions for month of Aug = 94,335

Drops
- Drop from June = 10%
- Drop from July = 26%

**Comment by benbrasso-agile6** [comment]
So, I looked at this a bit today and am basically validating Lori's data points above. I looked at more week-to-week through slightly different metrics, because I'm trying to understand if, or what, values in the new GA4 are best to use. E.g., how its calculating unique visitors vs non, etc., is all different. Comparing Universal GA metrics to GA4 metrics is apples to oranges; however, our DataDog metrics for completions are showing a similar trend as follows...

Anyways, here is sort of what I found while digging today. Again, which lines up with Lori's work.

- Previous iteration of pre-check-in that started with the form introduction page
    - Roughly 41k avg. non-unique pageviews of /complete PER WEEK
    - ~76% conversion rate from /introduction to /complete
- Current iteration of pre-check-in that starts from /appointments list page
    - Roughly $32k avg. non-unique pageviews of /complete PER WEEK
    - ~54% conversation rate from /appointments to /complete

So, basically, confirming what we know. But, I sort of had to do that work in order to know what to look at next.

So, next step would be to simply dive into the /appointments page and find out what users are selecting. Note: I did see some clicks on /my-health, which means some folks are using the accordion component and finding the link to review all of their appointments, because they expected to review them all.
If I were to make a giant leap hypothesis, I would say, potentially, getting the upcoming appts feature released sooner could potentially help in keeping the user in pre-check-in vs. having them abandon it and visiting Appointments on VA.gov (and can't get back to pre-check-in).

I'm going to create another ticket that's specific to looking more at what's happening on the /appointments page (with numbers, vs. just my cursory glance that I mentioned above).

[Pre-check-in landing page comparison.xlsx](https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBK25VQmc9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--1166ca6fac1ae12faa58adc01fb6fe0cc5e01398/Pre-check-in%20landing%20page%20comparison.xlsx)

https://docs.google.com/spreadsheets/d/1g7ONg8L3erW9_oGY6knRozCgma01DAiVx_jdVQ_zIRY/edit?gid=0#gid=0
